### ISSUE TITLE
a11y: card should not be focusable as a whole if it contains sub-elements

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#14248]
From a recent discussion:

"Don't focus the entire card as shown in the screenshot. A card usually contains a single entity of information that doesn't require the user to navigate or understand groupings at a more granular level. Since your UI will require this, focusing the card can actually cause focus to move around in unexpected ways when using "Explore by touch". Instead, ensure focus moves to elements within the card."

**Comment by goderbauer** [comment]
This cannot be expressed in the current system. What we want is Card to describe its SemanticsConfiguration as: "Children can pass semantics information on to me, but only if I don't end up with explicit child SemanticsNodes".

In other words: If card has any explicit SemanticsNodes, it needs to force all child semantics to be explicit.
